engine-types/src/types/mod.rs

```diff
@@ -1,21 +1,21 @@
-use crate::{str, vec, Address, String, Vec, U256};
+use crate::{str, vec, String, Vec, U256};
 
+pub mod address;
 pub mod balance;
 pub mod fee;
 pub mod gas;
 pub mod wei;
 
+pub use address::*;
 pub use balance::*;
 pub use fee::*;
 pub use gas::*;
 pub use wei::*;
 
-pub type RawAddress = [u8; 20];
 pub type RawU256 = [u8; 32];
-// Big-endian large integer type.
+/// Big-endian large integer type.
```

Comment:
I think this comment actually applies to `RawU256`, and `Unformatted binary data of fixed length.` applies to `RawH256`.